core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -599,21 +584,69 @@ public function show()
 							$label = $i;
 						}
 
-						// Always encode special characters (thanks to Oliver Klee)
```

Comment:
We should re-add this comment in line 640 so we do not lose the credits.

Reply:
Of course, restored in 8788aaa0b and b491472920aeec9d2a9253cf766fcbd902cf43c1.